docs/sources/datasources/prometheus/_index.md

```diff
@@ -28,7 +28,7 @@ For instructions on how to add a data source to Grafana, refer to the [administr
 Only users with the organization `administrator` role can add data sources and edit existing data sources.
 Administrators can also [configure the data source via YAML](#provision-the-data-source) with Grafana's provisioning system.
 
-Once you've added the Prometheus data source, you can [configure it][configure-prometheus-data-source] so that your Grafana instance's users can create queries in its [query editor]({{< relref "./query-editor" >}}) when they [build dashboards][build-dashboards], use [Explore][explore], and [annotate visualizations]({{< relref "./query-editor#apply-annotations" >}}).
+Once you've added the Prometheus data source, you can [configure it][configure-prometheus-data-source] so that your Grafana instance's users can create queries in its [query editor]({{< relref "./query-editor" >}}) when they [build dashboards][build-dashboards], use [Explore][explore], and [annotate visualizations](https://grafana.com/docs/grafana/latest/dashboards/build-dashboards/annotate-visualizations/).
```

Comment:
If you use `<GRAFANA_VERSION>` instead of `latest`, it will automatically be replaced with the correct version in "latest", "next", and any versioned releases.

That being said, this page is mounted in both Grafana and Grafana Cloud documentation and I believe this should instead use the `docs/reference` shortcode.

In Grafana Cloud the page should link to https://grafana.com/docs/grafana-cloud/visualizations/dashboards/build-dashboards/annotate-visualizations/.


You'll need the following suggestion and also to add some Markdown to the end of the file between the [`{{% docs/reference %}}` tags](https://github.com/grafana/grafana/pull/78673/files#diff-33bdf7b85528fdd40886418bb4acebeae7f122c62b65ad11a5b3c94b5a98c24bR170-R171):

```markdown
[annotate visualizations]: "/docs/grafana/ -> /docs/grafana/<GRAFANA_VERSION>/dashboards/build-dashboards/annotate-visualizations"
[annotate visualizations]: "/docs/grafana-cloud/ -> /docs/grafana/<GRAFANA_VERSION>/visualizations/dashboards/build-dashboards/annotate-visualizations"
```

```suggestion
Once you've added the Prometheus data source, you can [configure it][configure-prometheus-data-source] so that your Grafana instance's users can create queries in its [query editor]({{< relref "./query-editor" >}}) when they [build dashboards][build-dashboards], use [Explore][explore], and [annotate visualizations][].
```

Reply:
done